beacon-chain/rpc/beaconv1/config.go

```diff
@@ -23,5 +30,21 @@ func (bs *Server) GetSpec(ctx context.Context, req *ptypes.Empty) (*ethpb.SpecRe
 
 // GetDepositContract retrieves deposit contract address and genesis fork version.
 func (bs *Server) GetDepositContract(ctx context.Context, req *ptypes.Empty) (*ethpb.DepositContractResponse, error) {
-	return nil, errors.New("unimplemented")
+	ctx, span := trace.StartSpan(ctx, "beaconv1.GetDepositContract")
+	defer span.End()
+
+	buf := bytes.NewBuffer(params.BeaconConfig().GenesisForkVersion)
+	chainId, err := binary.ReadUvarint(buf)
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "could not obtain genesis fork version: %v", err)
+	}
+	byteAddress := [20]byte(bs.PowchainInfoFetcher.DepositContractAddress())
```

Comment:
Similar to my last feedback, we do not need to implement a new service getter to obtain deposit contract address. This is already accessible via `params.BeaconConfig().DepositContractAddress` (where powchain service gets it too)

Reply:
Good catch. I have to go through the config file and make a mental note of what is available.